lua/starfall/libs_sh/hook.lua

```diff
@@ -499,25 +584,6 @@ end
 -- @server
```

Comment:
`removed` hook says serverside only, but is actually shared. It was like that before, too, I'm just saying it here since you fixing it here would be easier than me making a PR and having you resolve the merge conflict.